modules/logging-set-input-rate-limit.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-flow-control-mechanisms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-set-input-rate-limit_{context}"]
+= Configuring log forwarder input rate limits
+
+You can limit the rate of incoming logs that are collected by configuring the `ClusterLogForwarder` custom resource (CR).
+Input limits can be set on a per-container or per-group basis.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+
+.Procedure
+
+. Add a `maxRecordsPerSecond` limit value to the `ClusterLogForwarder` CR for a specified input.
++
+The following examples shows how to configure input rate limits for different scenarios.
++
+.Example `ClusterLogForwarder` CR that sets a per-container limit for containers with certain labels
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+# ...
+  inputs:
+    - name: <input_name> <1>
+      application:
+        selector:
+          matchLabels: { example: label } <2>
+      limitPerContainer:
+          maxRecordsPerSecond: 0
+# ...
+----
+<1> The input name.
+<2> A list of labels. If these labels match labels that are applied to a container, the per-container limit specified in the `maxRecordsPerSecond` field is applied to that container.
```

Comment:
"applied to a pod"   

.... labels are associated with pods... but we are collecting log from containers

"... field is applied to those containers."